sdk/keyvault/keyvault-keys/test/utils/testAuthentication.ts

```diff
@@ -35,8 +34,11 @@ export async function authenticate(that: Context, version?: string): Promise<any
     env.AZURE_CLIENT_SECRET
   );
 
-  const keyVaultName = getKeyvaultName();
-  const keyVaultUrl = `https://${keyVaultName}.vault.azure.net`;
+  const keyVaultUrl = env.AZURE_KEYVAULT_URL;
+  if (!keyVaultUrl) {
+    throw new Error("Missing AZURE_KEYVAULT_URL environment variable.");
```

Comment:
After some thinking, I'm realizing that AZURE_KEYVAULT_URL is going to be missing since we didn't specify a default value during playback. This gets defined on this section above:

```ts
  const recorderEnvSetup: RecorderEnvironmentSetup = {
    replaceableVariables: {
      AZURE_CLIENT_ID: "azure_client_id",
      AZURE_CLIENT_SECRET: "azure_client_secret",
      AZURE_TENANT_ID: "azure_tenant_id",
      KEYVAULT_NAME: "keyvault_name"
    },
```